about.html

```diff
@@ -12,53 +12,73 @@
 </head>
 
 <body>
+  <header>
+    <div class="header">
+        <img src="img/lambda-black.png">
+        <nav>
+            <a href="#"> Home</a>
```

Comment:
a little oversight, you forgot to link the Home link to the index.html. So when the user navigate to the about.html page they are unable to get back using the navbar